other/APIs/client_secret.bcheck

```diff
@@ -0,0 +1,63 @@
+metadata:
+    language: v2-beta
+    name: "Use of Known API Variable"
+    description: "This BCheck looks for known API variables (client_secret, client_id, refresh_token, & APIKey)."
+    author: "Kyle Gilligan"
+    tags: "passive", "API", "client_secret", "client_id", "ApiKey"
+
+define:
+    # Issue details (for discovery of insecure API variables) as individual string texts.
+    issueDetail1 = `A known sensitive API parameter has been discovered within a front-end file of this web`
+    issueDetail1a = `application.`
+    issueDetail1FULL = `{issueDetail1} {issueDetail1a}`
```

Comment:
These are probably simple enough to merge into a single variable (no need for interpolated strings then as well).

Reply:
Made changes based on recommendation.

Reply:
Made changes based on recommendation.